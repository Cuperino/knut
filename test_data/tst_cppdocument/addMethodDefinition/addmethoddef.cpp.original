#include "addmethoddef.h"

Student::Student(const std::string& message)
    : m_message(message)
{}

int Student::foo(const std::string& s) {}
