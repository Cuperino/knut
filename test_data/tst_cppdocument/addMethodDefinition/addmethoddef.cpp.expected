#include "addmethoddef.h"

Student::Student(const std::string& message)
    : m_message(message)
{}

int Student::foo(const std::string& s) {}

bool Student::func(std::string s, int a) {}

bool Student::func(const QString &str) const {
std::cout << str.toStdString() << std::endl;
}
